pyannote/audio/interactive/common/utils.py

```diff
@@ -110,7 +109,13 @@ def remove_audio_before_db(examples: List[Dict]) -> List[Dict]:
 
 def get_chunks(source: Path, chunk_duration: Optional[float] = None):
 
-    files = ProdigyAudioLoader(source)
+    # Diff and review recipes can give only one file as source
+    if os.path.isdir(source):
+        files = ProdigyAudioLoader(source)
+    else:
+        name = os.path.basename(source).rsplit(".", 1)[0]
```

Comment:
Whenever possible, try to use `pathlib.Path` instances instead of calls to `os.path` unctions to make code easier to read.